src/effects/VST/VSTEffect.cpp

```diff
@@ -3594,6 +3595,18 @@ EffectSettings VSTEffect::MakeSettings() const
 
 VSTEffectValidator::~VSTEffectValidator() = default;
 
+
+VSTEffectValidator::VSTEffectValidator
+(
+   VSTEffectInstance&       instance,
+   EffectUIClientInterface& effect,
+   EffectSettingsAccess&    access,
+   wxWindow*                pParent
```

Comment:
I suggest it again:  at step 2, you don't need `effect`, `access`, `pParent` members because you can just use the protected members of the base classes of `VSTEffectValidator.`